### ISSUE TITLE
Bug when generating script for Get-ItemPropertyValue

### ISSUE THREAD

**Issue by Immakon** [rzander/Reg2CI#7]
When generating a script for something like a check for the bound SSL certificate in IIS the script doesn't work. the [byte[]] string is  formatted with extra ( and ' which breaks it.
Stripping it down to `[byte[]](string)`  fixes the issue.

`if((Get-ItemPropertyValue -LiteralPath 'HKLM:\SYSTEM\CurrentControlSet\Services\HTTP\Parameters\SslBindingInfo\0.0.0.0:443' -Name 'AppId' -ea SilentlyContinue) -join ',' -eq ('([byte[]]( string ))' -join ',')) {  }`
This was not a problem a while back.

**Comment by rzander** [comment]
can you please post the corresponding reg-file... THX

**Comment by Immakon** [comment]
This is the content of the reg file:
```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\HTTP\Parameters\SslBindingInfo\0.0.0.0:443]
"AppId"=hex:81,e1,c3,4d,4b,e1,21,4a,b0,22,59,fc,66,9b,09,14
```

**Comment by rzander** [comment]
Fixed in V1.0.3.8
Started a wiki page to collect test-cases: https://github.com/rzander/Reg2CI/wiki/Test-Reg-File
